scripts/drone/utils/windows_images.star

```diff
@@ -0,0 +1,12 @@
+"""
+This module contains all the windows docker images that are used to build test and publish Grafana.
+All the windows images needed to be in a different file than the other images, since they cannot be scanned
+by trivy. Related issue: https://github.com/aquasecurity/trivy/issues/1392
+"""
+
+windows_images = {
+    "windows_image": "mcr.microsoft.com/windows:1809",
```

Comment:
The prefix `windows_` feels redundant here 🙂

Reply:
Agreed!